src/containers/Progress/index.tsx

```diff
@@ -0,0 +1,227 @@
+        const milestones = milestoneResponse.data;
+
+        if (milestones.length > 0) {
+          // there should only be one milestone per repository at a time, get the latest one if there's multiple
+          const lastMilestone = milestones[0];
+          const issuesResponse = await githubApi.getIssuesForMilestone(repoName, lastMilestone.number);
+
+          if (teamMilestone) {
+            teamMilestone = {issues: {...issuesResponse.data, ...teamMilestone.issues}, milestone: lastMilestone};
+          } else {
+            teamMilestone = {issues: issuesResponse.data, milestone: lastMilestone};
+          }
+        }
+      });
+      await Promise.all(teamMilestonePromises);
+
+      if (teamMilestone) {
+        setter(teamMilestone);
+      }
+    } catch (err) {
+      // TBD on how to handle errors since there are so many different milestones
+    }
+  }
+
+  function getFilteredMilestone(milestones: Milestone[], title: string) {
+    return milestones.filter((milestone) => milestone.title.toLowerCase().includes(title.toLowerCase()))[0];
+  }
+
+  async function setMilestonesFromCommunityRepo() {
+    try {
+      const teamMilestoneDetail = teamMilestoneDetails.Community;
+      const repoName = teamMilestoneDetail.repositoryNames[0]; // assumption: community only has one repo
+      const milestoneResponse = await githubApi.getMilestones(repoName);
+      const milestones = milestoneResponse.data;
+      const audit = getFilteredMilestone(milestones, TeamName.audit);
+      const community = getFilteredMilestone(milestones, TeamName.community);
+      const general = getFilteredMilestone(milestones, 'Sprint');
+
+      const [auditIssues, communityIssues, generalIssues] = await Promise.all([
+        githubApi.getIssuesForMilestone(repoName, audit.number),
+        githubApi.getIssuesForMilestone(repoName, community.number),
+        githubApi.getIssuesForMilestone(repoName, general.number),
+      ]);
+
+      setAuditMilestone({issues: auditIssues.data, milestone: audit});
+      setCommunityMilestone({issues: communityIssues.data, milestone: community});
+      setGeneralMilestone({issues: generalIssues.data, milestone: general});
+    } catch (err) {
+      // TBD on how to handle errors since there are so many different milestones
+    }
+  }
+
+  async function setAllMilestones() {
+    setIsLoading(true);
+    await Promise.all([
+      getMilestone(TeamName.backEnd, (milestone) => setBackEndMilestone(milestone)),
+      getMilestone(TeamName.blockchain, (milestone) => setBlockchainMilestone(milestone)),
+      getMilestone(TeamName.design, (milestone) => setDesignMilestone(milestone)),
+      getMilestone(TeamName.devOps, (milestone) => setDevOpsMilestone(milestone)),
+      getMilestone(TeamName.frontEnd, (milestone) => setFrontEndMilestone(milestone)),
+      getMilestone(TeamName.marketing, (milestone) => setMarketingMilestone(milestone)),
+      setMilestonesFromCommunityRepo(),
```

Comment:
note that there's two separate functions used to get the milestone because:
- the Community Team, Audit Team and General Sprint is within the Community Repo. I am differentiating them based on the title for now. 
Community -> `Community` in milestone title
Audt -> `Audit` in milestone title
General -> `Sprint` in milestone title

-  the other milestones of other teams reside within their own repos

Reply:
got it.